backend/debug_task_breakdown.py

```
@@ -0,0 +1,93 @@
+    
+    # Test with a simple request
+    message = "Add a dark mode toggle button to the header"
+    
+    print("🔍 Debugging AI Response")
+    print("=" * 50)
+    print(f"Request: {message}")
+    print("\nCalling AI model...")
+    
+    try:
+        # Get the raw response from Gemini
+        system_prompt = agent._generate_task_breakdown.__code__.co_consts[1]  # This won't work, let me try a different approach
```

Ruff (0.12.2)

Local variable `system_prompt` is assigned to but never used

Remove assignment to unused variable `system_prompt`

(F841)